lib/git_diff_parser/patch.rb

```diff
@@ -1,7 +1,7 @@
 module GitDiffParser
   # Parsed patch
   class Patch
-    RANGE_INFORMATION_LINE = /^@@ .+\+(?<line_number>\d+),/
+    RANGE_INFORMATION_LINE = /^@@ -(?<old_line_number>\d+),.+\+(?<line_number>\d+),/
```

Comment:
From what I can, tell `,\d+` should be optional.

For example:

```diff
@@ -1 +1 @@
-type Query { field: String! }
+type Query { field2: String! }
```

I believe we can change this line to:

```ruby
RANGE_INFORMATION_LINE = /^@@ -(?<old_line_number>\d+)(,\d+)? \+(?<line_number>\d+)(,\d+)?/
```

I wrote a failing test and fix here: https://github.com/cjoudrey/ruby-git_diff_parser/commit/858a9457851718ff178ee74172dac0b9ea22c943